packages/shared/src/lib/analytics.ts

```diff
@@ -144,6 +144,7 @@ export enum AnalyticsEvent {
   // Reading Streaks
   OpenStreaks = 'open streaks',
   DismissStreaksMilestone = 'dismiss streaks milestone',
+  ScheduleReadingStreakReminder = 'schedule reading streak reminder',
```

Comment:
@idoshamun It's now a engineering decision, this works for you, basically copy of reading reminder, but for streaks specific

Reply:
There wasn't anything in the spec, so I was going to ask about it, But I see you beat me to it.

Reply:
Good call my suggestion would be:

```suggestion
  ScheduleReadingStreakReminder = 'schedule streak reminder',
```

Reply:
Changed it